atom/browser/net/url_request_context_getter.cc

```diff
@@ -341,22 +342,37 @@ net::URLRequestContext* URLRequestContextGetter::GetURLRequestContext() {
       host_resolver = std::move(remapped_resolver);
     }
 
-    net::HttpAuthPreferences auth_preferences;
+    std::vector<std::string> supported_schemes = {
+      net::kBasicAuthScheme,
+      net::kDigestAuthScheme,
+#if defined(USE_KERBEROS)
+      net::kNegotiateAuthScheme,
+#endif
+      net::kNtlmAuthScheme
+    };
+    http_auth_preferences_ =
+        std::make_unique<net::HttpAuthPreferences>(supported_schemes
+#if defined(OS_POSIX)
+                                                   ,
+                                                   std::string()
```

Comment:
This is some odd whitespace, could it be changed to `, std::string()` all on one line?

Reply:
That just how clang-format likes it, this is one of those case where clang-format and cpplint disagree